src/main/java/com/socrata/datasync/job/IntegrationJob.java

```diff
@@ -71,6 +71,28 @@
     private boolean publishViaDi2Http = false;
     private ControlFile controlFile = null;
 
+    // Ok, so this is awful.  What we're trying to do is change the
```

Comment:
😵 I read this several times, but I'm still unsure how we preserve existing behavior.
> "before that change" is defined as "not having a flag that says it's from after that change"
Does that mean calling `loadOldSijFile` is already preserving existing behavior?